Gems/Atom/RHI/DX12/Code/Source/RHI/CommandList.cpp

```diff
@@ -490,6 +491,19 @@ namespace AZ
             }
         }
 
+        void CommandList::Submit(FfxFsr2Context& context, FfxFsr2DispatchDescription& fsr2DispatchItem)
```

Comment:
I think the FfxFsr2DispatchDescription& can be const since the fsr api is 
`FFX_API FfxErrorCode ffxFsr2ContextDispatch(FfxFsr2Context* context, const FfxFsr2DispatchDescription* dispatchDescription);`